src/test/scala/example/BookBot.scala

```diff
@@ -0,0 +1,60 @@
+package example
+
+import org.scalatest._
+import io.circe._, io.circe.parser._
+
+class BookBotSpec extends FunSpec with Matchers {
+  val isbn10: String = "0545790352"
+  val isbn13: String = "0545790352123"
+  val jsonString: String = raw"""
+    {
+      "volumeInfo": {
+        "title": "Harry Potter and the Sorcerer's Stone",
+        "authors": [
+          "J. K. Rowling"
+        ],
+        "industryIdentifiers": [
+          {
+            "type": "ISBN_10",
+            "identifier": "$isbn10"
+          },
+          {
+            "type": "ISBN_13",
+            "identifier": "$isbn13"
+          }
+        ]
+      }
+    }
+  """
+
+  describe("BookBot") {
+    describe("getBook") {
+      it("gets a response from the api") {
+        assert(BookBot.getBook("harry potter").isInstanceOf[String])
```

Comment:
I feel like this test is a bit superfluous because it only asserts the `type` that it's returning, which is already asserted by the the type signature of `BookBot.getBook`. I'd like to see this test testing the behavior of `BookBot,getBook` and not the return type.

Reply:
Do you have any recommendations for Stubbing/Mocking/Test fixturing the http package so that i can right more meaningful tests? From what I've seen most people (including the package im using) are making a mock server and running against that.

Reply:
We use scalamock at our client for mocking things that we want to be called but not actually run: https://scalamock.org/quick-start/. 

Essentially we assert that the mocked object gets called with the correct parameters, and then give it a fixture/stub for what it's supposed to return if all goes well. But in terms of mocking third party libraries you'd want to create a simple wrapper around the http service, and then make a mock of that. 

I've also used `MockFactory` in a previous project. Here's a file where I'm testing out my http routes using mocks and fixtures: https://github.com/lpenzey/omnibus/blob/master/src/test/scala/com/lpenzey/routes/v1/BusRoutesSpec.scala